xml/tuning_cgroups.xml

```diff
@@ -221,8 +220,177 @@
+  </para>
+  <para>
+      &sle; ships with a custom configuration that overrides the upstream
+      default. 
+      <filename>/usr/lib/systemd/system.conf.d/20-suse-defaults.conf</filename>
+      overrides the default limit with the following configuration:
+  </para>
+  <screen>
+[Manager]
+DefaultTasksMax=infinity
+  </screen>
+  <para>
+      When you query the value with <command>systemctl</command> it looks like
+      this:
+  </para>
+  <screen>&prompt.user;systemctl show --property DefaultTasksMax
+DefaultTasksMax=18446744073709551615</screen>
+  <para>
+       This is the same as having no limit. It is not a requirement to change the 
+       default, but setting some limits may help to prevent system crashes
+       from runaway processes.
+   </para>
+   <para>
+       Start by disabling the &sle; default by linking 
+       <filename>20-suse-defaults.conf</filename> to 
+       <filename>/dev/null</filename>, and putting the link in
+       <filename>/etc/systemd/system.conf.d/</filename>. First create
+       <filename>system.conf.d/</filename> if it does not exist, then 
+       create the link:
+   </para>
+   <screen>&prompt.root;mkdir /etc/systemd/system.conf.d/
+&prompt.root;ln -s /dev/null /etc/systemd/system.conf.d/20-suse-defaults.conf
+   </screen>
+   <para>
+       Then change the global default by creating a new override
+       file, <filename>/etc/systemd/system.conf.d/50-tasksmax.conf</filename>,
+       and write the following lines to set new global limit of 256 tasks per 
+       systemd unit:
+   </para>
+   <screen>
+[Manager]
+DefaultTasksMax=256
+</screen>
+   <para>
+       Load the new setting, then verify that it changed:
+   </para>
+   <screen>&prompt.root;systemctl daemon-reload
+&prompt.user;systemctl show --property DefaultTasksMax
+DefaultTasksMax=256
+   </screen>
+   <para>
+       You may adjust this value to suit your needs. Now you can set higher 
+       limits on individual services as needed. This example is for MariaDB. 
+       First check the current active value:
+   </para>
+   <screen>
+&prompt.user;systemctl status mariadb.service
+  ● mariadb.service - MariaDB database server
+   Loaded: loaded (/usr/lib/systemd/system/mariadb.service; disabled; vendor pr>
+   Active: active (running) since Tue 2020-05-26 14:15:03 PDT; 27min ago
+     Docs: man:mysqld(8)
+           https://mariadb.com/kb/en/library/systemd/
+ Main PID: 11845 (mysqld)
+   Status: "Taking your SQL requests now..."
+    Tasks: 30 (limit: 256)
+   CGroup: /system.slice/mariadb.service
+           └─11845 /usr/sbin/mysqld --defaults-file=/etc/my.cnf --user=mysql
+   </screen>
+   <para>
+       The Tasks line shows that MariaDB currently has 30 tasks running, and has
+       an upper limit of 256. The following example demonstrates how to raise
+       MariaDB's limit to 8096. Create a new override file,
+       <filename>/etc/systemd/system.conf.d/30-mariadb.conf</filename>, 
```

Comment:
To override the limit for MariaDB only, the drop-in file needs to be located in `/etc/systemd/system/mariadb.service.d`.